pyaardvark/cli_tool.py

```diff
@@ -59,6 +59,37 @@ def i2c_rd(a, args):
     data = a.i2c_master_read(args.i2c_address, args.num_bytes)
     print_hex(data)
 
+
+def i2c_slave_str(a, addr):
```

Comment:
```suggestion
def i2c_slave_detect(a, addr):
```

Also please only return True or False here and put the ' ' and the ' --' into `i2c_scan()`